ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__rosa-sts-test-rhods-operator-4.13-lp-rosa.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+  openshift-cli-installer:
+    name: openshift-cli-installer
+    namespace: ci
+    tag: latest
+  rhods-addon-tests:
+    name: ods-ci
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tests:
+- as: rosa-sts-rhods-test-operator
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CLUSTER1_CONFIG: name=rosa-sts;platform=rosa;region=us-east-2;version=4.13;channel-group=stable;compute-machine-type=m5.4xlarge;timeout=90m;replicas=6;fips=false;ocm-env=stage
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "operator-install", "failure_type": "all", "classification": "Provisioning - Operators", "jira_project": "LPTOCPCI"},
+          {"step": "cluster-install", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster (ROSA)", "jira_project": "LPTOCPCI"},
+          {"step": "cluster-uninstall", "failure_type": "all", "classification": "Infrastructure Deprovisioning - Cluster (ROSA)", "jira_project": "LPTOCPCI"}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      OPENSHIFT_VERSION: "4.13"
+      OPERATOR1_CONFIG: name=rhods-operator;namespace=redhat-ods-operator
+      S3_BUCKET_NAME: openshift-cli-installer
+      S3_BUCKET_PATH: openshift-ci
+    workflow: operator
```

Comment:
The cluster install time in the latest passing job looks a bit long
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/43065/rehearse-43065-pull-ci-rhpit-interop-tests-main-rosa-sts-test-rhods-operator-4.13-lp-rosa-rosa-sts-rhods-test-operator/1704871878830592000#1:build-log.txt%3A22

Looking at the table here provisioning time for rosa classic should be about 40 minutes
https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html-single/install_rosa_with_hcp_clusters/index#rosa-hcp-classic-comparison_rosa-hcp-sts-creating-a-cluster-quickly

We see the install times matching with the table [here ](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/43386/rehearse-43386-periodic-ci-rhpit-interop-tests-main-web-terminal-4.13-lp-rosa-sts-web-terminal-aws-rosa-sts/1704179597332975616#1:build-log.txt%3A25)when deploying rosa classic using the [rosa-aws-sts workflow](https://steps.ci.openshift.org/workflow/rosa-aws-sts) in this [PR](https://github.com/openshift/release/pull/43386)

Do you know if these install times have varied?

Reply:
@ascerra I think this question is above the scope of this PR, this scenario is simply using an automation which might change the provisioning time (overall- including steps before provisioning).

Reply:
Also, this scenario is not using the `rosa-aws-sts` workflow. This is a part of the configuration I was asked to successfully pass.

Reply:
@ascerra the installer makes sure that the cluster is ready and `osd-cluster-ready` ends successfully so your comparison should be to a complete flow. 
BTW, may have missed it but I do not see https://steps.ci.openshift.org/workflow/rosa-aws-sts wait for this flow.

BTW - https://steps.ci.openshift.org/reference/rosa-cluster-wait-ready-operators by itself waits for 30 minutes.

Reply:
The [rosa-cluster-wait-ready-operators](https://steps.ci.openshift.org/reference/rosa-cluster-wait-ready-operators) is a part of the chain https://steps.ci.openshift.org/chain/rosa-cluster-provision.

I've seen it take [15 min here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/43386/rehearse-43386-periodic-ci-rhpit-interop-tests-main-web-terminal-4.13-lp-rosa-sts-web-terminal-aws-rosa-sts/1704179597332975616#1:build-log.txt%3A29) but I'm sure this varies and can very easily take 30 minutes.

We can resolve this comment, the time stood out at first glance but thats just because this new install is packing a lot of these steps into one.